Tests/Unit/Controller/RedirectControllerTest.php

```diff
@@ -45,22 +40,22 @@ protected function setUp(): void
     {
         $this->controller = new WebsiteRedirectController();
 
-        $this->request = $this->prophesize(Request::class);
         $this->queryBag = class_exists(InputBag::class) ? new InputBag() : new ParameterBag();
         $this->redirectRoute = $this->prophesize(RedirectRouteInterface::class);
-
-        $this->request->reveal()->query = $this->queryBag;
     }
 
     public function testRedirect()
     {
         $target = '/test';
         $statusCode = 301;
 
+        $request = Request::create($target);
+        $request->query = $this->queryBag;
```

Comment:
would call `$request->query->set` and not create `this->queryBag` longer

Reply:
ahhh now i see what you mean :D !